animations/lib/src/basics/05_animated_builder.dart

```diff
@@ -15,7 +15,7 @@ class _AnimatedBuilderDemoState extends State<AnimatedBuilderDemo>
     with SingleTickerProviderStateMixin {
   static const Color beginColor = Colors.deepPurple;
   static const Color endColor = Colors.deepOrange;
-  Duration duration = Duration(milliseconds: 800);
+  Duration duration = Duration(milliseconds: 1000);
```

Comment:
Any particular reason for this change? It doesn't seem to relate to the title changes.

Reply:
Thanks for reviewing PR.
Yes, it is not related to title, But because of long duration user can see more clearly that how colors are changing. 
Not a big change but a bit slow animation creates huge impact.